.github/workflows/playwright-test-reusable.yml

```diff
@@ -22,14 +27,22 @@ on:
     secrets:
       CURRENTS_RECORD_KEY:
         required: true
+      QA_PERFORMANCE_METRICS_WEBHOOK_URL:
+        required: true
+      QA_PERFORMANCE_METRICS_WEBHOOK_USER:
+        required: true
+      QA_PERFORMANCE_METRICS_WEBHOOK_PASSWORD:
+        required: true
 
 env:
   PLAYWRIGHT_BROWSERS_PATH: packages/testing/playwright/ms-playwright-cache
   NODE_OPTIONS: --max-old-space-size=3072
   # Disable Ryuk to avoid issues with Docker since it needs privileged access, containers are cleaned on teardown anyway
   TESTCONTAINERS_RYUK_DISABLED: true
   PLAYWRIGHT_WORKERS: 3 # We have 2 CPUs on this runner but we can use more workers since it's low CPU intensive
-
+  QA_PERFORMANCE_METRICS_WEBHOOK_URL: ${{ secrets.QA_PERFORMANCE_METRICS_WEBHOOK_URL }}
+  QA_PERFORMANCE_METRICS_WEBHOOK_USER: ${{ secrets.QA_PERFORMANCE_METRICS_WEBHOOK_USER }}
+  QA_PERFORMANCE_METRICS_WEBHOOK_PASSWORD: ${{ secrets.QA_PERFORMANCE_METRICS_WEBHOOK_PASSWORD }}
```

Comment:
Not sure if we need to define these in the "global" env if they are only used in single action down the line